### DISCUSSION TITLE
Explanation of Debug page?

### DISCUSSION THREAD

**Discussion by mcdent** [blakeblackshear/frigate#2532] [Ask A Question]
Is there an explanation of the debug page of frigate anywhere? I want to figure out if my cameras and detection is working well or if the CPU etc is keeping up with things? I don't have a Coral (can't find one cheaper than a kidney at the moment) so am unsure if I can get away with adding my final camera :)
![Screenshot 2021-12-27 at 22 09 39](https://user-images.githubusercontent.com/8769856/147510483-7906a0ce-f0bc-4d9c-a3f9-4e85190ecc6c.jpg)

**Comment by blakeblackshear** [comment]
The main thing to watch for is skipped fps. If that goes above 0, frigate is skipping frames to try and keep up. Other than that, just take a look at CPU usage.

**Comment by Jensilein** [comment]
Hi, how can I activate that debug viewing page? I have set debugging in frigate.yml but don‘t see the debug viewer in the GUI.

**Reply by NickM-27** [reply]
It's now called system page.

**Comment by Cold-Lemonade** [comment]
@mcdent why are you running an old version of Frigate?

**Reply by NickM-27** [reply]
@andrew-lemon it was not old at the time this was posted
